apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -46,7 +46,10 @@ local function new_redis_cluster(conf)
                         .. " err: " .. err
         end
 
-        config.serv_list[i] = {ip = host, port = port}
+        config.serv_list[i] = {
+            ip = host, 
+            port = port
```

Comment:
we'd better add a `,` at the end of line

Reply:
ok